genai-perf/genai_perf/config/generate/genai_perf_config.py

```diff
@@ -88,6 +93,18 @@ def get_obj_args(self) -> Namespace:
 
         return obj_args
 
+    ###########################################################################
+    # Representation Methods
+    ###########################################################################
+    def representation(self) -> Representation:
```

Comment:
Right now it's just a string, because it is based on the CLI, but there is no guarantee that is what it will be going forward once we move to a config based interface.

Reply:
In that case, we should either:
1. need a different name for the type (e.g. more specific to its use case something like `CommandString`). In Python world, the word "representation" is established enough to make me expect the return type to be a string (see [Python `repr` function](https://docs.python.org/3/library/functions.html#repr)). So if we expect the underlying type to be something different than `str`, then we should probably avoid using `Representation` as a name for the type.
2. hold-off the change until the change to config based interface actually happens.

Reply:
Understood, I've reverted this back to `str`